bundles/org.openhab.binding.doorbird/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,224 @@
+	</channel-type>
+	<channel-type id="motionImage">
+		<item-type>Image</item-type>
+		<label>Motion Detected Image</label>
+		<description>Image when motion was last detected</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="light">
+		<item-type>Switch</item-type>
+		<label>Light</label>
+		<description>Energize the light relay</description>
+		<state pattern="%s"></state>
+	</channel-type>
+	<channel-type id="openDoor1">
+		<item-type>Switch</item-type>
+		<label>Open Door 1</label>
+		<description>Energize opendoor / alarm output relay 1</description>
+		<state pattern="%s"></state>
+	</channel-type>
+	<channel-type id="openDoor2">
+		<item-type>Switch</item-type>
+		<label>Open Door 2</label>
+		<description>Energize opendoor / alarm output relay 2</description>
+		<state pattern="%s"></state>
+	</channel-type>
+	<channel-type id="getImage">
+		<item-type>Switch</item-type>
+		<label>Get Image</label>
+		<description>Get an image</description>
+		<state pattern="%s"></state>
+	</channel-type>
+	<channel-type id="imageTimestamp">
+		<item-type>DateTime</item-type>
+		<label>Image Timestamp</label>
+		<description>Time when image was captured from device</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="image">
+		<item-type>Image</item-type>
+		<label>Image</label>
+		<description>Image from device</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="doorbellHistoryIndex">
+		<item-type>Number</item-type>
+		<label>Doorbell History Index</label>
+		<description>Index of historical image for doorbell press</description>
+		<state min="1" max="50" step="1"></state>
+	</channel-type>
+	<channel-type id="doorbellHistoryTimestamp">
+		<item-type>DateTime</item-type>
+		<label>Doorbell History Timestamp</label>
+		<description>Time when doorbell was pressed for history image</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="doorbellHistoryImage">
+		<item-type>Image</item-type>
+		<label>Doorbell History Image</label>
+		<description>Historical image for doorbell press</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="motionHistoryIndex">
+		<item-type>Number</item-type>
+		<label>Motion History Index</label>
+		<description>Index of Historical image for motion</description>
+		<state min="1" max="50" step="1"></state>
+	</channel-type>
+	<channel-type id="motionHistoryTimestamp">
+		<item-type>DateTime</item-type>
+		<label>Motion History Timestamp</label>
+		<description>Time when motion was detected for history image</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="motionHistoryImage">
+		<item-type>Image</item-type>
+		<label>Motion History Image</label>
+		<description>Historical image for motion sensor</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="getDoorbellMontage" advanced="false">
```

Comment:
Missing in the docs? Can it be a thing action? Or `RefreshType.REFRESH`?

Reply:
Removed channel and will use REFRESH on the associated image channel.